### PR TITLE
Added Azure DevOps CI/CD definition

### PR DESCRIPTION
**Pull request by ivan-nosar**
This PR introduces the following changes:
- Configure `azure-pipelines.yml` file with AzDO build definition
- Change device set used for cucumber tests
- Remove Jenkins and Travis CI configurations
- Update calabash.framework (from `0.21.7` to `0.21.8`)
- Update Xcode project settings

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        echo "Xcode select path: $(xcode-select --print-path)"
+      displayName: "Select Xcode $(XCODE_VERSION)"
+
+    - script: git submodule update --init --recursive
+      displayName: "Update submodules"
+
+    - script: |
+        set -e
+        gem update --system
+        gem uninstall -Vax --force --no-abort-on-dependent calabash-cucumber
```
Comment (maxim-lobanov) [inline azure-pipelines.yml]: I think we don't need to do that because ADO VM is empty by default

**Comment by maxim-lobanov** [comment]
@ivan-nosar , I see that you have moved files from `bin/ci/jenkins`→ `bin/ci/azure-pipelines`.
May be it makes sense to put them to the `bin/ci` instead of specifying CI-dependent folder.

**Comment by ivan-nosar** [comment]
> @ivan-nosar , I see that you have moved files from `bin/ci/jenkins`→ `bin/ci/azure-pipelines`.
> May be it makes sense to put them to the `bin/ci` instead of specifying CI-dependent folder.

Done

File: azure-pipelines.yml (hunk 1 above)
Comment (ivan-nosar) [reply azure-pipelines.yml]: Done. I've left only updating of gems

File: CalWebViewApp/bin/make/ipa-cal.sh [hunk 2]
```diff
@@ -127,7 +127,7 @@ fi
 cat >"${XTC_DIR}/Gemfile" <<EOF
 source "https://rubygems.org"
 
-gem "calabash-cucumber", "2.99.0"
+gem "calabash-cucumber"
```
Comment (jmoody) [inline CalWebViewApp/bin/make/ipa-cal.sh]: This must be 2.99.0.  It is the last allowed version in Test Cloud.

File: CalWebViewApp/ios-webview-test-app.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -284,10 +284,11 @@
 				};
 			};
 			buildConfigurationList = F52BFB071987EF8700299AEF /* Build configuration list for PBXProject "ios-webview-test-app" */;
-			compatibilityVersion = "Xcode 3.2";
```
Comment (jmoody) [inline CalWebViewApp/ios-webview-test-app.xcodeproj/project.pbxproj]: Wow. :)

File: CalWebViewApp/ios-webview-test-app.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -561,7 +563,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (jmoody) [inline CalWebViewApp/ios-webview-test-app.xcodeproj/project.pbxproj]: Good. The deployment target for all products should be iOS 9.

File: CalWebViewApp/ios-webview-test-app.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jmoody) [inline CalWebViewApp/ios-webview-test-app.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Remove this from the git index.  Add an entry in the .gitignore.

File: azure-pipelines.yml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        echo "Xcode version: $(xcrun xcodebuild -version)"
+        echo "Xcode path: $(xcrun -f xcodebuild)"
+        echo "Xcode select path: $(xcode-select --print-path)"
+      displayName: "Select Xcode $(XCODE_VERSION)"
+
+    - script: git submodule update --init --recursive
+      displayName: "Update submodules"
+
+    - script: gem update --system
+      displayName: "Update gems"
```
Comment (jmoody) [inline azure-pipelines.yml]:
```suggestion
      displayName: "Update rubygems version"
```

File: README.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 | master  |  [license](LICENSE) |
 |---------|---------------------|
-| [![Build Status](https://travis-ci.org/calabash/ios-webview-test-app.svg?branch=master)](https://travis-ci.org/calabash/ios-webview-test-app)<br>[![Build Status](http://calabash-ci.xyz:8081/job/ios-webview-test-app/job/feature%252Fjenkins_pipeline/badge/icon)](http://calabash-ci.xyz:8081/job/ios-webview-test-app/job/feature%252Fjenkins_pipeline/)| [![License](https://img.shields.io/badge/licence-MIT-blue.svg)](http://opensource.org/licenses/MIT) |
+| [![Build Status](https://msmobilecenter.visualstudio.com/Mobile-Center/_apis/build/status/calabash.ios-webview-test-app?branchName=master)](https://msmobilecenter.visualstudio.com/Mobile-Center/_build/latest?definitionId=3606&branchName=master)| [![License](https://img.shields.io/badge/licence-MIT-blue.svg)](http://opensource.org/licenses/MIT) |
```
Comment (jmoody) [inline README.md]: Great.

File: CalWebViewApp/bin/make/ipa-cal.sh (hunk 2 above)
Comment (ivan-nosar) [reply CalWebViewApp/bin/make/ipa-cal.sh]: Latest version available at the moment is `0.21.10`.

File: CalWebViewApp/ios-webview-test-app.xcodeproj/project.pbxproj (hunk 3 above)
Comment (ivan-nosar) [reply CalWebViewApp/ios-webview-test-app.xcodeproj/project.pbxproj]: Yeah :)

File: CalWebViewApp/ios-webview-test-app.xcodeproj/project.pbxproj (hunk 4 above)
Comment (ivan-nosar) [reply CalWebViewApp/ios-webview-test-app.xcodeproj/project.pbxproj]: Done

File: CalWebViewApp/ios-webview-test-app.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 5 above)
Comment (ivan-nosar) [reply CalWebViewApp/ios-webview-test-app.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Done

File: azure-pipelines.yml (hunk 6 above)
Comment (ivan-nosar) [reply azure-pipelines.yml]: Fixed

**Pull request merged**